config/config.go

```diff
@@ -19,7 +19,8 @@ import (
 )
 
 type Config struct {
-	Systemd Systemd
+	Systemd    Systemd
+	WriteFiles []File `yaml:"write_files"`
```

Comment:
Please omit this for now. This will live in a different part of the global config.